deploy/crds/worker-0.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: worker-0-bmc-secret
+type: Opaque
+data:
+  username: YWRtaW4=
+  password: cGFzc3dvcmQ=
+
+---
+apiVersion: metalkube.org/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: worker-0
+spec:
+  online: true
+  bmc:
+    address: ipmi://192.168.122.1:6233/?mac=00:73:49:3a:76:8e;net=provisioning
```

Comment:
What are `mac=..` and `net=..` parts doing here? If it's for creating ports, why not have a separate field?

Reply:
It is indeed for the port. See https://github.com/metalkube/metalkube-docs/blob/master/design/bmc-address.md for details about using the URL. So far we only need MAC and net values for libvirt, so I will eventually create a libvirt "type" but the type parsing isn't there, yet.

Reply:
@dtantsur, the latest patch in the pull request shows what I intend for those extra values.

Reply:
So just to clarify, the argument is that the UI will be simpler if we encode the port details in the BMC URL string?

I'm really not clear how that can be the case, since it means we'll either need some special validation, or the UI can't validate the fields at all?

The BMC connection is logically separate from the PXE provisioning mac/network so combining them seems weird to me, but perhaps I'm missing some context from previous discussions.